contracts/utils/math/Math.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-// OpenZeppelin Contracts (last updated v5.0.0) (utils/math/Math.sol)
+// OpenZeppelin Contracts (last updated v5.0.2) (utils/math/Math.sol)
```

Comment:
This is going to be updated automatically during the release process, we should not manually touch it.
```suggestion
// OpenZeppelin Contracts (last updated v5.0.0) (utils/math/Math.sol)
```